src/Components/Facility/Investigations/InvestigationTable.tsx

```diff
@@ -173,12 +168,13 @@ export const InvestigationTable = ({
                               };
                         handleValueChange(value, target);
                       }}
+                      className="print:text-black"
                     />
                   );
                 })
               ) : (
-                <tr className="text-center text-secondary-500">
-                  No tests taken
+                <tr className="text-center text-secondary-500 print:text-black">
+                  <td className="col-span-6">t(no_tests_taken)</td>
```

Comment:
You missed quotes and brackets here.

```suggestion
                  <td className="col-span-6">{t("no_tests_taken")}</td>
```